cpp/src/community/nvgraph_gdf.cu

```diff
@@ -200,10 +197,9 @@ void analyzeClustering_edge_cut_nvgraph(Graph* gdf_G,
                                                   gdf_column* clustering,
                                                   float* score) {
 
-  CUGRAPH_EXPECTS(gdf_G != nullptr, "Invalid API parameter");
-  CUGRAPH_EXPECTS(gdf_G->adjList != nullptr, "Invalid API parameter");
-  CUGRAPH_EXPECTS(clustering != nullptr, "Invalid API parameter");
-  CUGRAPH_EXPECTS(clustering->data != nullptr, "Invalid API parameter");
+  CHECK_GRAPH(gdf_G);
+  CUGRAPH_EXPECTS(clustering != nullptr, "Invalid API parameter: clustering is NULL");
+  CUGRAPH_EXPECTS(clustering->data != nullptr, "Invalid API parameter: clustering data is NULL");
   CUGRAPH_EXPECTS(!clustering->valid, "Column must be valid");
```

Comment:
I think error message here is misleading. This says clustering column should be non-nullable (so, valid should be nullptr).

Reply:
I fully agree with your statement.  I'm just trying to improve what is there and try to make it less misleading.    This will be reworked when gdf_column is removed